code/modules/food_and_drinks/pizzabox.dm

```diff
@@ -350,4 +350,4 @@
 	var/obj/item/pizza_type = pizza_preferences[noms.ckey]
 	pizza = new pizza_type (src)
 	var/obj/item/organ/tongue/T = noms.getorganslot(ORGAN_SLOT_TONGUE)
-	pizza.foodtype = T.liked_food //it's our favorite!
+	pizza.foodtype = T?.liked_food //it's our favorite!
```

Comment:
Will this cause problems if pizza.foodtype is set to null?

Reply:
I'm not sure if food type on a food item could ever be null